server/remote/common.go

```diff
@@ -16,11 +20,13 @@ type DateTime struct {
 
 const RFC3339NanoNoTimezone = "2006-01-02T15:04:05.999999999"
 
-func NewDateTime(t time.Time) *DateTime {
+// NewMicrosoftDateTime creates a DateTime that is compatible with Microsoft's API.
+func NewMicrosoftDateTime(t time.Time, timeZone string) *DateTime {
```

Comment:
_really_ not sure, but in the caller context, woudn't `remote.NewDateTime()` be sufficiently intention-revealing?